plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.tsx

```diff
@@ -94,39 +95,49 @@ const useStyles = makeStyles((theme: BackstageTheme) =>
   }),
 );
 
-const countEntitiesBy = (
+const listEntitiesBy = (
   entities: Array<Entity>,
   kind: EntitiesKinds,
   type?: EntitiesTypes,
 ) =>
-  entities.filter(
-    e => e.kind === kind && (type ? e?.spec?.type === type : true),
-  ).length;
+  entities
+    .filter(e => e.kind === kind && (type ? e?.spec?.type === type : true))
+    .map(e => e.metadata.name);
+
+const countEntitiesBy = (
+  entities: Array<Entity>,
+  kind: EntitiesKinds,
+  type?: EntitiesTypes,
+) => listEntitiesBy(entities, kind, type).length;
 
 const EntityCountTile = ({
   counter,
   className,
+  entities,
   name,
 }: {
   counter: number;
   className: EntitiesTypes;
+  entities: string[];
   name: string;
 }) => {
   const classes = useStyles();
   return (
-    <Box
-      className={`${classes.card} ${classes[className]}`}
-      display="flex"
-      flexDirection="column"
-      alignItems="center"
-    >
-      <Typography className={classes.bold} variant="h6">
-        {counter}
-      </Typography>
-      <Typography className={classes.bold} variant="h6">
-        {name}
-      </Typography>
-    </Box>
+    <Tooltip title={`${entities.join(', ')}`} arrow>
```

Comment:
I'm thinking it's best to add some sensible max count at which point it falls back to filling in with `...`, unless the tooltip already handles that